Src/WitsmlExplorer.Frontend/components/QueryEditor.tsx

```diff
@@ -29,6 +30,7 @@ export const QueryEditor = (props: QueryEditorProps) => {
       editor.renderer.$cursorLayer.element.style.display = "none";
     } else {
       editor.completers = [customCompleter];
+      editor.renderer.on("afterRender", (_: any, renderer: any) => updateLinesWithWidgets(editor, renderer));
```

Comment:
I recommend fixing the types, cleaning up `any` types later can be tedious.
Suggested fix:
```js
import { Ace } from "ace-builds";

  const onLoad = (editor: Ace.Editor) => {
    editor.renderer.setPadding(10);
    editor.renderer.setScrollMargin(10, 0, 0, 0);
    if (readonly) {
      editor.renderer.hideCursor();
    } else {
      editor.completers = [customCompleter];
      editor.renderer.on("afterRender", (_: string, renderer: Ace.VirtualRenderer) => updateLinesWithWidgets(editor, renderer));
    }
  };
```
Could you please verify if the method `hideCursor()` is the correct one to use?

Reply:
We had a discussion and decided to skip this task because it was complicated due to issues with the package.